readthedocs/rtd_tests/tests/test_project.py

```diff
@@ -193,6 +195,21 @@ def test_get_latest_build_excludes_external_versions(self):
         # Test that External Version is not considered for get_latest_build.
         self.assertEqual(self.pip.get_latest_build(), None)
 
+    def test_git_provider_name_github(self):
+        self.assertEqual(self.pip.git_provider_name, GITHUB_BRAND)
```

Comment:
These should probably explicitly assign the `repo`, like the `gitlab` tests, so they won't fail if we change something elsewhere.

Reply:
Done!